lib/handler.js

```diff
@@ -49,7 +49,7 @@ exports = module.exports = function createStrongErrorHandler(options) {
     if (!err.status && !err.statusCode && res.statusCode >= 400)
       err.statusCode = res.statusCode;
 
-    var data = buildResponseData(err, isDebugMode);
+    var data = buildResponseData(err, options);
```

Comment:
With this change in place, the variable `isDebugMode` is no longer used and should be removed.